persistence/src/commonMain/db_global/com/wire/kalium/persistence/Accounts.sq

```diff
@@ -0,0 +1,51 @@
+import com.wire.kalium.persistence.dao.QualifiedIDEntity;
+import com.wire.kalium.persistence.model.LogoutReason;
+
+CREATE TABLE Accounts (
```

Comment:
since this only returns id and logout reason, is it right to be called accountInfo?